src/music.c

```diff
@@ -321,9 +323,9 @@ void SDLCALL music_mixer(void *udata, Uint8 *stream, int len)
                 int fade_steps = music_playing->fade_steps;
 
                 if (music_playing->fading == MIX_FADING_OUT) {
-                    volume = (volume * (music_volume * (fade_steps-fade_step))) / (fade_steps * MIX_MAX_VOLUME);
+                    volume = (music_volume * (fade_steps-fade_step)) / fade_steps;
```

Comment:
The initialization of volume a few lines above, i.e. `int volume = Mix_MasterVolume(-1);`
has no point any more: the initialization needs to go away.

Reply:
Forgot to remove here (at my side I already did that), thanks for the otice